apps/src/gamelab/TextConsole.jsx

```diff
@@ -0,0 +1,169 @@
+/** @file Text display over sprite lab visualization column */
+import PropTypes from 'prop-types';
+import React from 'react';
+import {Transition} from 'react-transition-group';
+
+const lineHeight = 18;
+const maxHeight = lineHeight * 6;
+
+export const styles = {
+  hide: {
+    display: 'none'
+  },
+  console: {
+    display: '',
+    background: 'rgba(200,200,200,0.5)',
+    position: 'absolute',
+    top: 0,
+    left: 0,
+    zIndex: 2,
```

Comment:
Is it necessary to use zIndexes here? I believe you should be able to achieve this same result by simply rendering `TextConsole` _after_ `ProtectedVisualizationDiv` in `GameLabVisualizationColumn`

Reply:
Fixed. Thank you